mozregression/history.py

```diff
@@ -0,0 +1,46 @@
+"""
+Representation of the bisection history.
+"""
+
+
+class BisectionStep(object):
```

Comment:
Could this be a namedtuple? https://docs.python.org/2.7/library/collections.html#namedtuple-factory-function-for-tuples-with-named-fields